src/table/pagination-table/components/head/TableHeadWrapper.tsx

```diff
@@ -28,7 +28,7 @@ function TableHeadWrapper() {
 
   useEffect(() => {
     if (headRowRef.current) {
-      setHeadRowHeight(headRowRef.current.getBoundingClientRect().height);
+      setHeadRowHeight(headRowRef.current.clientHeight);
```

Comment:
Why change to `clientHeight`? Seems like the `getBoundingClientRect` returns a more suitable value as you get a fractional value. Which may fit better with "WYSIWYG"

Reply:
Actually I got a bit help from Max here. 
The issue happened when Totals row was set. If I remember it correctly, getBoundingClientRect returns rendering sizes which, beside having a Totals row, a kind of size ratio miscalculation occurred when I added scrollTop adjustment So the adjustment was applied on Totals row as the first visible row! Also resizing the table when snapshot was used ended a big gap between header and Totals rows. Using clientHeight solved all these.

Reply:
Ok then I won't look into that anymore 👍